docs/pages/installation.mdx

```diff
@@ -12,6 +12,20 @@ guide](./get-started.mdx).
 
 </Admonition>
 
+<Admonition type="warning" title="Teleport client compatibility">
```

Comment:
The successive Admonitions appear pretty busy to me:
<img width="965" alt="image" src="https://user-images.githubusercontent.com/25250668/236039530-f67021f3-7f70-48c2-8f2f-550813831fc3.png">

I think we can strip both `Admonition` wrappers and add an intro sentence so the text flows more smoothly, making the first part of the guide looks like this:

```markdown
This guide shows you how to install Teleport binaries on your platform,
including:
- `teleport`
- `tsh`
- `tctl`
- `tbot`

If you are new to Teleport, we recommend following our [getting started 
guide](http://localhost:3000/docs/get-started/).

For best results, Teleport clients (tsh, tctl, tbot) should be the same major
version as the cluster they are connecting to. Teleport servers are compatible
with clients that are on the same major version or one major version older. 
Teleport servers do not support clients that are on a newer major version.

See our [Upgrading](http://localhost:3000/docs/management/operations/upgrading/) guide for more information.
```

Reply:
Added in [b8c4361](https://github.com/gravitational/teleport/pull/25548/commits/b8c4361eec5b876a390b42cb53974b2ddca0c991)